frontend-react/src/resources/AuthResource.ts

```diff
@@ -26,12 +26,6 @@ export default class AuthResource extends Resource {
     };
 
     static getBaseUrl = () => {
-        if (window.location.origin.includes("localhost"))
-            return "http://localhost:3000";
-        else if (window.location.origin.includes("staging"))
-            return "https://staging.prime.cdc.gov";
-        else if (window.location.origin.includes("test"))
-            return "https://test.reportstream.cdc.gov";
-        else return "https://prime.cdc.gov";
+        return process.env.REACT_APP_BASE_URL;
```

Comment:
nice

Reply:
yeah, it felt good removing that if/elseif block.